test/Functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require_once '../functions.php';
+
+use PHPUnit\Framework\Testcase;
+
+class Functions extends TestCase
+{
+    public function testSuccessPrintInsect()
+    {
+        $input = [
+            'id' => 1,
+            'common_name' => 'Jewel Bug',
+            'species' => 'Scutiphora Pedicellata',
+            'date_spotted' => '2018-03-15',
+            'location_spotted' => 'India',
+            'size' => '5-20 mm',
+            'image_path' => 'images/jewel_bug.jpeg'];
+        $expectedOutput = '<div class="insects"><img src="images/jewel_bug.jpeg"><h2>Jewel Bug</h2>' .
+            '<p class="species">Scutiphora Pedicellata</p><p>Date spotted: 2018-03-15</p>' .
+             '<p>Location spotted: India</p><p>Size: 5-20 mm</p></div>';
+        $actualOutput = printInsect($input);
+        $this->assertEquals($expectedOutput, $actualOutput);
+    }
+    public function testFailurePrintInsect()
+    {
+        $input = ['id' => '1', 'name' => 'Jewel Bug', 'species' => 'Scutiphora Pedicellata', 'date_spotted' => '2018-03-15', 'location_spotted' => 'India', 'size' => '5-20 mm', 'image_path' => 'images/jewel_bug.jpeg'];
+        $expectedOutput = 'I\'m sorry, that array doesn\'t contain the expected keys';
```

Comment:
this sentence should have . in order to match exactly with the error message that the function returns